techniques/css/C29.html

```diff
@@ -348,19 +344,19 @@ <h4>The JavaScript components</h4>
 }</code></pre>
 			</section>
 			
-			<p class="working-example">A working example of this code, <a href="../../working-examples/script-change-css-property/">Using a client-side JavaScript to change a CSS property</a>, is available.</p>
+			<p class="working-example">このコードの実装例 <a href="../../working-examples/script-change-css-property/">Using a client-side JavaScript to change a CSS property</a> が利用可能である。</p>
```

Comment:
working example の訳を、他のテクニックと合わせて「動作例」としました。
```suggestion
			<p class="working-example">このコードの動作例 <a href="../../working-examples/script-change-css-property/">Using a client-side JavaScript to change a CSS property</a> が利用可能である。</p>
```